API-Server/src/main/java/swm/hkcc/LGTM/app/modules/serverDrivenUI/ServerDrivenContent.java

```diff
@@ -0,0 +1,51 @@
+package swm.hkcc.LGTM.app.modules.serverDrivenUI;
+
+import lombok.Builder;
+import lombok.Getter;
+import swm.hkcc.LGTM.app.modules.mission.dto.MissionDetailsDto;
+import swm.hkcc.LGTM.app.modules.mission.dto.MissionDto;
+import swm.hkcc.LGTM.app.modules.mission.dto.MissionTitleDto;
+
+@Builder
+@Getter
+public class ServerDrivenContent<T> {
+
+    private ViewType viewType;
+
+    private Theme theme;
+
+    private T content;
+
+    public static ServerDrivenContent from(MissionDto missionDto, Theme theme, ViewType viewType) {
+        return ServerDrivenContent.builder()
+                .viewType(viewType)
+                .theme(theme)
+                .content(missionDto)
+                .build();
+    }
+
+    public static ServerDrivenContent from(MissionDetailsDto missionDetailsDto, Theme theme, ViewType viewType) {
+        return ServerDrivenContent.builder()
+                .viewType(viewType)
+                .theme(theme)
+                .content(missionDetailsDto)
+                .build();
+    }
+
+    public static ServerDrivenContent from(MissionTitleDto missionTitleDto, Theme theme, ViewType viewType) {
+        return ServerDrivenContent.builder()
+                .viewType(viewType)
+                .theme(theme)
+                .content(missionTitleDto)
+                .build();
+    }
```

Comment:
serverDrivenUI 패키지 내에 Mission...Dto를 파라미터로 받는 동작이 어색하게 느껴집니다.
제네릭 T에 대한 생성 메소드인 만큼 Object를 파라미터로 지정하는게 더 좋지 않을까요??

```suggestion
    public static ServerDrivenContent from(Object content, Theme theme, ViewType viewType) {
        return ServerDrivenContent.builder()
                .viewType(viewType)
                .theme(theme)
                .content(content)
                .build();
    }

```

Reply:
좋은 것 같습니다! 코드 반영해서 커밋하겠습니다!